content/es/_index.html

```diff
@@ -0,0 +1,60 @@
+---
+title: "Orquestación de contenedores para producción"
+abstract: "Automatización del despliegue, escalado y administración de contenedores"
+cid: home
+---
+
+{{< deprecationwarning >}}
+
+{{< blocks/section id="oceanNodes" >}}
+{{% blocks/feature image="flower" %}}
+### Kubernetes (K8s) es una plataforma de código abierto para automatizar la implementación, el escalado y la administración de aplicaciones en contenedores.
+
+Kubernetes agrupa los contenedores que conforman una aplicación en unidades lógicas para una fácil administración y descubrimiento. Kubernetes se basa en [15 años de experiencia en la ejecución de cargas de trabajo de producción en Google](http://queue.acm.org/detail.cfm?id=2898444), combinada con las mejores ideas y prácticas de la comunidad.
+{{% /blocks/feature %}}
+
+{{% blocks/feature image="scalable" %}}
+#### Preparado para trabajar a escala
+
+Diseñado sobre los mismos principios que le permiten a Google ejecutar miles de millones de contenedores cada semana, Kubernetes puede escalar sin aumentar su equipo de operaciones.
+
+{{% /blocks/feature %}}
+
+{{% blocks/feature image="blocks" %}}
+#### Preparado para cualquier complejidad
+
+Ya sea para pruebas en local o para cargas de trabajo para una empresa global, la flexibilidad de Kubernetes crece contigo para entregar sus aplicaciones de manera consistente y fácil sin importar la complejidad.
```

Comment:
s/pruebas en local/pruebas locales

Reply:
Fixed in https://github.com/kubernetes/website/pull/13543/commits/bd9afbbbfecd2e24628c525f1d166134bd18dc0e